File: app/src/main/java/me/hachy/routineweek/model/Todo.java
package me.hachy.routineweek.model;

import java.util.Date;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

public class Todo extends RealmObject {

    @PrimaryKey
    public long id;

    public int day;

    public String content;

    public boolean done;

    public Date createdTime;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public int getDay() {
        return day;
    }

    public void setDay(int day) {
        this.day = day;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public boolean isDone() {
        return done;
    }

    public void setDone(boolean done) {
        this.done = done;
    }

    public Date getCreatedTime() {
        return createdTime;
    }

    public void setCreatedTime(Date createdTime) {
        this.createdTime = createdTime;
    }
}

File: app/src/main/java/me/hachy/routineweek/model/Todo.kt
package me.hachy.routineweek.model

import java.util.Date

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Todo : RealmObject() {

    @PrimaryKey
    var id: Long = 0

    var day: Int = 0

    var content: String? = null

    var done: Boolean = false

    var tagColor: Int = 0

    var createdTime: Date? = null
}
